docs/deployment/uninstall-kubeflow.md

```diff
@@ -0,0 +1,36 @@
++++
+title = "Uninstall Kubeflow"
+description = "Delete Kubeflow deployments and Amazon EKS clusters"
+weight = 80
++++
+
+## Uninstall Kubeflow on AWS
+
+First, delete all existing Kubeflow profiles. 
+
+```bash
+kubectl get profile
+kubectl delete profile --all
+```
+
+Then, delete the Kubeflow deployment with the following command:
+
+```bash
+kustomize build example | kubectl delete -f 
```

Comment:
You can delete a Kubeflow deployment by running the `kubectl delete` command on the manifest according to the deployment option you chose. For example, to delete a vanilla installation, run the following command:

`kustomize build docs/deployment/vanilla/ | kubectl delete -f -`

This command assumes you have the repository in the same state as when you installed Kubeflow

Reply:
@judyheflin you can replace lines 16 to 20 with the content Suraj posted here